apps/services/utils.py

```diff
@@ -0,0 +1,84 @@
+from apps.common.exceptions import BackendException
+from apps.users.services import student_list_service
+
+from collections import defaultdict
+from fastapi import UploadFile, status as http_status
+from sqlalchemy.ext.asyncio import AsyncSession
+from typing import Callable, DefaultDict, Dict, List, Set, Tuple
+import xlrd
+
+
+def check_file_content_type(file: UploadFile) -> None:
```

Comment:
returned type = None or BackendException

Reply:
That function return nothing and pointing BackendException is not right. BackendEcxeption raising, but not returning

Reply:
got it, agreed